actions/ci-lint-go/files/golangci-default.yml

```diff
@@ -0,0 +1,156 @@
+version: "2"
+linters:
+  enable:
+    - containedctx
+    - copyloopvar
+    - depguard
+    - errname
+    - errorlint
+    - exhaustive
+    - fatcontext
+    - ginkgolinter
+    - gomoddirectives
+    - gosec
+    - loggercheck
+    - mirror
+    - misspell
+    - noctx
+    - perfsprint
+    - prealloc
+    - revive
+    - rowserrcheck
+    - spancheck
+    - sqlclosecheck
+    - testifylint
+    - unconvert
+    - whitespace
+  settings:
+    depguard:
+      rules:
+        main:
+          list-mode: lax
+          deny:
+            - pkg: cosmossdk.io/errors
+              desc: Use the standard library instead
+            - pkg: github.com/go-gorm/gorm
+              desc: Use github.com/jmoiron/sqlx directly instead
+            - pkg: github.com/gofrs/uuid
+              desc: Use github.com/google/uuid instead
+            - pkg: github.com/pkg/errors
+              desc: Use the standard library instead, for example
+                https://pkg.go.dev/errors#Join
+            - pkg: github.com/satori/go.uuid
+              desc: Use github.com/google/uuid instead
+            - pkg: github.com/test-go/testify/assert
+              desc: Use github.com/stretchr/testify/assert instead
+            - pkg: github.com/test-go/testify/mock
+              desc: Use github.com/stretchr/testify/mock instead
+            - pkg: github.com/test-go/testify/require
+              desc: Use github.com/stretchr/testify/require instead
+            - pkg: go.uber.org/multierr
+              desc: Use the standard library instead, for example
+                https://pkg.go.dev/errors#Join
+            - pkg: gopkg.in/guregu/null.v1
+              desc: Use gopkg.in/guregu/null.v4 instead
+            - pkg: gopkg.in/guregu/null.v2
+              desc: Use gopkg.in/guregu/null.v4 instead
+            - pkg: gopkg.in/guregu/null.v3
+              desc: Use gopkg.in/guregu/null.v4 instead
```

Comment:
nit/ I have always found this format difficult to read - does yaml allow any other style that might make the list of package imports stand out more? e.g. obviously this doesn't work, but as an unrealistic ideal:
```yaml
            - cosmossdk.io/errors: Use the standard library instead
            - github.com/go-gorm/gorm: Use github.com/jmoiron/sqlx directly instead
            - github.com/gofrs/uuid: Use github.com/google/uuid instead
```

Reply:
Yeah definitely

Reply:
Updated in cb5cfc9d